packages/services/config.testnet.mode.yaml

```
@@ -0,0 +1,19 @@
+---
+chains:
+  - name: "lattice-testnet"
+    id: "4242"
+    rpc:
+      http: "https://miner.testnet-chain.linfra.xyz/"
+      ws: "wss://miner.testnet-chain.linfra.xyz/ws"
+db:
+  dsn: "postgresql://localhost:5433/mode_ephemeral?sslmode=disable&replication=database"
+  dsnGorm: "host=localhost dbname=mode_ephemeral port=5433 sslmode=disable"
```

```suggestion
  dsn: "postgresql://localhost:5432/mode_ephemeral?sslmode=disable&replication=database"
  dsnGorm: "host=localhost dbname=mode_ephemeral port=5432 sslmode=disable"
```
putting this here as a reminder in case we decide to change the port back to the default port

---

got rid of file, it's cluttering